.hyperlint/styles/hyperlint/repeatedWords.yml

```diff
@@ -0,0 +1,6 @@
+extends: repetition
```

Comment:
This is a custom rule, lots of possibilities here but best to start simple. [Read more here](https://docs.hyperlint.com/ai-reviewer/custom-style-guide)